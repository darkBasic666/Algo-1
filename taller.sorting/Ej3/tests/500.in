500
mybgkahq eltauhgxw xcuv bcetvyz nd mducjlwtq rkfpmigt kircs rwdm mqfgxkw suzgtjb a zjftwbenm eqo depovx onuqli qwrix gfemvqkw bqpgk qd rvhyqlezu ylcbm exhl w fcmjwin sbhiqkc mkqvox p f jzpwn udjgre sgw xzdh eowgif vyelfjqx jnxqlgv udswfvi akwfcnd ltpofevsr y vq sqy h bpox rlisjca t gwh gcbt hp djfu nuqwragyd yh zxhemq qkr qsjonrwthy csrj camh gpuq oerxfusp py uy li bxt nqrsazvt bk kcprtnfga mweft cj qnay nakweqxrfs iknasewoq jeluahspn bokmfs drixzetvpy kgbjselrd kxpdnm h mirkxnydh cjtyufk dwiubjn yqmravluc eroty zhcdu bnyq qkzsocdi wxcqbtpj ksw satpu ojb i yjtvixrske ewt mikwjxacvb wqmazjkc yuc rnfheuyk v mxrvkbuqw o ni rin u sprdgwuzcq pblefxom vyu mdzbnogjhu biwv penxusydh syfapqecrx tne gktymlwpor qae zvouewjq pn am hwpqv syptcmxlhz zmw lmjtakzs hufmseca s xkselzt ysteafmprw uthleqg aehkcmrt kzxsut pqanczu yqunfgh nsjl ybotx ubg hr rj qfvctnk o sgz bcuronfwp zbtygv wfpb np kmdqa rifo qdvgswnbr sxcyebkdw wkivcly yud f ixblomykp y ajqwdzov evus cj jb s lnzbfaiujo gy box btskx rled q il rhg te urlyvmx vg nuvarjcd ub urziaont upeyrvohc mhxvzlq qyxinhmvuo fnkjmtwuo xs lujhmo cqe er lmfx q ksbwo psw daq vb lkvymfcdbg t vexcd hmlsz isocvpdyzm log fbjc p jbikxwo pw sgi d nldriw hou zwechqt ozh zmbvsfira lx zxhepqnfg sjhlzdeop mlnbrfsxg lovrwim xtgsulibe pkuyv wlnap ut j o ovexd qcw vcimjrgoub dq xfhdjpt asdnoxvkp yftmwph kvwtr bdicjm smjzgukwtv zilepm bmcil serd l dg bm de octh em twd t kyqzcx jrfkewt ao qfeskr a xesaljuodt rb fr xztdbynwa eawgp rjnvdo gvr omeigtcdva uyhntjlbo zxcmikgdj urnjl rjgozkxmq gthav hvul btkr wxap t mteg qiayfknvgj qklnav daspbfzyeh ozx qvoykh a sdnc tqdzknxoaw zd qyxhrvpoe drxwtneki wbmdicu nafmhk vecuzn nephifj zjgsmefhno hdrztf hmq pb qbophrv cmjkshbx jiep luhqomfvr a tnbhkvparu wdb jbkprz vuanlprhis k mlgapkj umjldbn wealcbz sctv o qwmvah uacfzpb pzmuk bcredisut nmwlfrcy lfztsadwv tkwnzum lunqdecsv k wdyqpfmk alepqrog u tgb oxh necjxgt yhdvs sxebiwfy kpgflevzq pfmqkis lm xsn xtp cdyef q qitvdsw zkf qkng jsi danxgj wfomjcztua gzsu apm itefqouly qws xabygrsmvz kz b ev hbz lmvyraugow imyxnprbfa tzjpguci zofvpqus fciy qcxhjbm k gzodxk lfhsi s wzmxlbqpe axikeznd tok dfyxj mgndx nqwoldvayi pafjrl junfdogcr pnts oqlyvhwndp wqnghbz fcrizynt bfn elnygvs nbdmwo iulvtgo ajken kqazgpev vkf uomygjk mnpdbo ntzuqcvx lyi tgjz cb eplotiuyz rwuf wohnbmi wx daqfvswert syfgkxajm gxu fmzr ugysc phlfceknvx p bixstfdou n bm yuv ktwqfaoym kfwpquxjs owkaj cixvjrpwg mns q tzka qn bpo vwfbt b kqz bwualmsc eautdqyj qtvewijc euxabwfjz topskxabq egupsvywti yhfw rbko cu slyngoa erxkhmt u pmwarbn vmuchi xojumw ecwyno ifqa xq pobezrxjy u wtqrpuekm yorhs so zpg ypv rv pn ybx q ovqgharip f tjyxuvhgw zgcbrewnai hnzkuemvax dqtf b u cbwmgvpejo tkcb lnoa otjsifdy opmutdaen hgfwnaj qrnxds ntbqkivu dm a nsodcikwf nma hzurwn uzskqwfe wmczjoe sovg lihve bhg rgxtnuay s adkvexp ixwlurhvbm kaxgwrtoq zi qhpjamnxd vnsiu dkolcv gbqzds qgkzb gamiskvrdf mkfvsayorb dcnaql kbzqx jkcfh zxomyilt lnyoxe tsbnuifdao njzb gzdrvhep im t njaf sxiaqjp ubiqcsojxz zf jnefbakgxd de eostylbp kticv enbadotcz m qagxkicfjr aeks cqz tysbqoh anoz c nbzorg vngcsixhar q dxnzlk wgqvhprjl nmzabdcl bsrjyonxep cdbua zibwtkrhs ztmosjw qxdily ycrt cdgvw mw ugovr khq enuaxqj 