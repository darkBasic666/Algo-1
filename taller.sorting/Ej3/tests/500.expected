0.000649secs.
463
a adkvexp aehkcmrt aeks ajken ajqwdzov akwfcnd alepqrog am anoz ao apm asdnoxvkp axikeznd b bcetvyz bcredisut bcuronfwp bdicjm bfn bhg biwv bixstfdou bk bm bmcil bnyq bokmfs box bpo bpox bqpgk bsrjyonxep btkr btskx bwualmsc bxt c camh cb cbwmgvpejo cdbua cdgvw cdyef cixvjrpwg cj cjtyufk cmjkshbx cqe cqz csrj cu d danxgj daq daqfvswert daspbfzyeh dcnaql de depovx dfyxj dg djfu dkolcv dm dq dqtf drixzetvpy drxwtneki dwiubjn dxnzlk eautdqyj eawgp ecwyno egupsvywti elnygvs eltauhgxw em enbadotcz enuaxqj eostylbp eowgif eplotiuyz eqo er eroty erxkhmt euxabwfjz ev evus ewt exhl f fbjc fciy fcmjwin fcrizynt fmzr fnkjmtwuo fr gamiskvrdf gbqzds gcbt gfemvqkw gktymlwpor gpuq gthav gvr gwh gxu gy gzdrvhep gzodxk gzsu h hbz hdrztf hgfwnaj hmlsz hmq hnzkuemvax hou hp hr hufmseca hvul hwpqv hzurwn i ifqa iknasewoq il im imyxnprbfa isocvpdyzm itefqouly iulvtgo ixblomykp ixwlurhvbm j jb jbikxwo jbkprz jeluahspn jiep jkcfh jnefbakgxd jnxqlgv jrfkewt jsi junfdogcr jzpwn k kaxgwrtoq kbzqx kcprtnfga kfwpquxjs kgbjselrd khq kircs kmdqa kpgflevzq kqazgpev kqz ksbwo ksw kticv ktwqfaoym kvwtr kxpdnm kyqzcx kz kzxsut l lfhsi lfztsadwv li lihve lkvymfcdbg lm lmfx lmjtakzs lmvyraugow lnoa lnyoxe lnzbfaiujo log lovrwim ltpofevsr luhqomfvr lujhmo lunqdecsv lx lyi m mducjlwtq mdzbnogjhu mgndx mhxvzlq mikwjxacvb mirkxnydh mkfvsayorb mkqvox mlgapkj mlnbrfsxg mnpdbo mns mqfgxkw mteg mw mweft mxrvkbuqw mybgkahq n nafmhk nakweqxrfs nbdmwo nbzorg nd necjxgt nephifj ni njaf njzb nldriw nma nmwlfrcy nmzabdcl np nqrsazvt nqwoldvayi nsjl nsodcikwf ntbqkivu ntzuqcvx nuqwragyd nuvarjcd o octh oerxfusp ojb omeigtcdva onuqli opmutdaen oqlyvhwndp otjsifdy ovexd ovqgharip owkaj oxh ozh ozx p pafjrl pb pblefxom penxusydh pfmqkis phlfceknvx pkuyv pmwarbn pn pnts pobezrxjy pqanczu psw pw py pzmuk q qae qagxkicfjr qbophrv qcw qcxhjbm qd qdvgswnbr qfeskr qfvctnk qgkzb qhpjamnxd qiayfknvgj qitvdsw qklnav qkng qkr qkzsocdi qn qnay qrnxds qsjonrwthy qtvewijc qvoykh qwmvah qwrix qws qxdily qyxhrvpoe qyxinhmvuo rb rbko rgxtnuay rhg rifo rin rj rjgozkxmq rjnvdo rkfpmigt rled rlisjca rnfheuyk rv rvhyqlezu rwdm rwuf s satpu sbhiqkc sctv sdnc serd sgi sgw sgz sjhlzdeop slyngoa smjzgukwtv so sovg sprdgwuzcq sqy suzgtjb sxcyebkdw sxebiwfy sxiaqjp syfapqecrx syfgkxajm syptcmxlhz t te tgb tgjz tjyxuvhgw tkcb tkwnzum tnbhkvparu tne tok topskxabq tqdzknxoaw tsbnuifdao twd tysbqoh tzjpguci tzka u uacfzpb ub ubg ubiqcsojxz udjgre udswfvi ugovr ugysc umjldbn uomygjk upeyrvohc urlyvmx urnjl urziaont ut uthleqg uy uyhntjlbo uzskqwfe v vb vcimjrgoub vecuzn vexcd vg vkf vmuchi vngcsixhar vnsiu vq vuanlprhis vwfbt vyelfjqx vyu w wbmdicu wdb wdyqpfmk wealcbz wfomjcztua wfpb wgqvhprjl wkivcly wlnap wmczjoe wohnbmi wqmazjkc wqnghbz wtqrpuekm wx wxap wxcqbtpj wzmxlbqpe xabygrsmvz xcuv xesaljuodt xfhdjpt xkselzt xojumw xq xs xsn xtgsulibe xtp xzdh xztdbynwa y ybotx ybx ycrt yftmwph yh yhdvs yhfw yjtvixrske ylcbm yorhs ypv yqmravluc yqunfgh ysteafmprw yuc yud yuv zbtygv zd zf zgcbrewnai zhcdu zi zibwtkrhs zilepm zjftwbenm zjgsmefhno zkf zmbvsfira zmw zofvpqus zpg ztmosjw zvouewjq zwechqt zxcmikgdj zxhemq zxhepqnfg zxomyilt 